src/app/admin/createUser/CreateUserForm.tsx

```diff
@@ -59,7 +64,7 @@ const getServerErrorMessage = (serverError: InviteUserError): string => {
     }
 };
 
-const formSections = [AccountDetails, UserRoleCard, UserDetailsCard];
+const formSections = [AccountDetails, UserDetailsCard, UserRoleCard];
 
 const CreateUserForm: React.FC = () => {
     const [fields, setFields] = useState(initialFieldValues);
```

Comment:
Maybe make these names a bit clearer e.g. userFields? Same for the initial field values

Reply:
I think there's only one field in each file, so these names won't be hard to distinguish?